parachain/pallets/basic-channel/src/inbound/test.rs

```diff
@@ -31,8 +31,8 @@ frame_support::construct_runtime!(
 		NodeBlock = Block,
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
-		System: frame_system::{Pallet, Call, Storage, Event<T>},
-		BasicInboundChannel: basic_inbound_channel::{Pallet, Call, Storage, Event<T>},
+		System: frame_system::{Pallet, RuntimeCall, Storage, RuntimeEvent<T>},
```

Comment:
Looks like the tests are complaining about the Runtime prefixes here.

```suggestion
		System: frame_system::{Pallet, Call, Storage, Event<T>},
```